app/routes/users/components/UserProfile.js

```diff
@@ -160,8 +188,17 @@ export default class UserProfile extends Component<Props> {
                 </div>
               )}
           </div>
-
           <div className={styles.rightContent}>
+            {isMe && (
+              <div>
+                <h3>Dine kommende arrangementer</h3>
```

Comment:
How about simply doing `{isMe && upcomingEvents.length && ( ...` here, instead of doing the same check later and returning EmptyState in UpcomingEvents? That would remove the "Dine kommende arrangementer" altogether when you have no upcoming events, which seems better to me. Would let you remove a bit of code as well.

Reply:
I did it that way because it was consistent with how it was done in the Feed. Actually I think that if the feed is showing when there is nothing there, I don't see a problem with the upcoming events staying there if empty. Just for consistency.

Reply:
I see what you mean, but I think the Feed EmptyState is there to make the page feel less empty. Having _two_ such emptystates doesn't really help solve that same issue. 

However, at least initially it might be a good idea to keep what you've made just so that people see that the feature has been added. We can remove it later if we decide that's best, but let's keep it for now. The visibility might be enough reason to keep it long term too.

Would you mind adding a bit of margin between the feed and the upcoming events though? :P The event list items have some margin built in so it mainly looks too tight when both lists are empty, but adding some margin for both cases wouldn't hurt.

Reply:
Agree on the thought that people can see more easily that the feature has been added if it stays. Increased the margin :)